galaxy/worker/importers/role.py

```diff
@@ -162,14 +182,18 @@ def _add_dependencies(self, role, dependencies):
                     role.dependencies.add(dep_role)
                     new_deps.append(dep)
                 except Exception as e:
-                    self.log.error(
-                        u"Error loading dependencies {}".format(e))
+                    msg = u'Error loading dependencies {}'.format(e)
+                    self.linter_data['linter_rule_id'] = 'dependency_load'
+                    self.linter_data['rule_desc'] = msg
+                    self.log.warning(msg, extra=self.linter_data)
```

Comment:
This was changed from error to warning since it doesn’t stop the importer from importing